drivers/sensor/isl29035/isl29035_trigger.c

```diff
@@ -16,6 +16,32 @@ extern struct isl29035_driver_data isl29035_data;
 
 LOG_MODULE_DECLARE(ISL29035, CONFIG_SENSOR_LOG_LEVEL);
 
+static inline void setup_int(struct device *dev,
```

Comment:
`setup_int` as a name is a bit confusing. I believe in case of other sensors that were updated we called this function `setup_irq`.

Reply:
In other cases we called this function `setup_foo` where `foo` is whatever name the signal was associated with.  In the original case of hts221 was `drdy` and for sht3xd was `alert`.

I've been trying to use the name from the datasheet in the devicetree property, but neither that nor `irq` is being done consistently.  In this case, the devicetree name is `int`.